test/torch/tensors/test_precision.py

```diff
@@ -693,6 +693,16 @@ def test_dtype():
     )
 
 
+def test_ndim():
+    x = torch.rand(2, 3, 4).fix_prec()
+    assert x.ndim == 3
+
+
+def test_T():
+    x = torch.rand(2, 3, 4).fix_prec()
+    assert x.T.shape == torch.Size([4, 3, 2])
```

Comment:
Test the value for transpose

Reply:
done in a new commit, by testing `x.fix_prec().T==x.T.fix_prec()`